libfwupd/fwupd.map

```diff
@@ -800,3 +800,35 @@ LIBFWUPD_1.8.3 {
     fwupd_security_attr_remove_flag;
   local: *;
 } LIBFWUPD_1.8.2;
+
+LIBFWUPD_1.8.4 {
+  global:
+    fwupd_bios_attr_add_possible_value;
+    fwupd_bios_attr_array_from_variant;
+    fwupd_bios_attr_copy;
+    fwupd_bios_attr_from_json;
+    fwupd_bios_attr_from_variant;
+    fwupd_bios_attr_get_current_value;
+    fwupd_bios_attr_get_description;
```

Comment:
I think all these need some pretty epic self tests

Reply:
Yeah, but was waiting until we figure out what to do about thinklmi not following the spec in case this needs major overhaul for assumption.

Reply:
done